src/libs/actions/Report.ts

```diff
@@ -682,6 +682,20 @@ function updateGroupChatAvatar(reportID: string, file?: File | CustomRNImageMani
     API.write(WRITE_COMMANDS.UPDATE_GROUP_CHAT_AVATAR, parameters, {optimisticData, failureData, successData});
 }
 
+/**
+ * Clear error and pending fields for the report avatar
+ */
+function clearAvatarErrors(reportID: string) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {
+        errorFields: {
+            avatar: null,
+        },
+        pendingFields: {
+            avatar: null,
+        },
```

Comment:
Why do we need to clear the pending fields?

Reply:
I don't see the need to clear the pendingFields. You can check the commit that added that to the existing `clearAvatarErrors` functions to see if this is something we want here

Reply:
Let's omit it then, we don't have a reason to clear the pending fields